pkg/sql/sem/tree/create_logical_replication.go

```diff
@@ -49,7 +49,7 @@ func (node *CreateLogicalReplicationStream) Format(ctx *FmtCtx) {
 	ctx.FormatNode(&node.Into)
 
 	if !node.Options.IsDefault() {
-		ctx.WriteString(" WITH ")
+		ctx.WriteString(" WITH OPTIONS ")
```

Comment:
You may need to rewrite the parser test, since it should fail now that "OPTION" is going to be written.

Reply:
Oh I think this actually won't work since there is not a parser case for it. Might need to do
ctx.WriteString(" WITH OPTIONS (")
ctx.FormatNode(&node.Options)
ctx.WriteString(")")

since there is a case for WITH OPTIONS ( logical_replication_options_list).

Reply:
yeah that's definitely correct. we need the parens.